src/main/java/core/basesyntax/Draw.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Draw {
+    String draws();
```

Comment:
```suggestion
    void draw();
```

Reply:
Why void? This method should return that what was "drawed". Explain please